docs/app/pages/components/components-sections/panels/side-panel/side-panel.component.html

```diff
@@ -190,6 +194,9 @@ <h3>Side Panel</h3>
     <tr uxd-api-property name="focusOnShow" type="boolean" defaultValue="false">
         Whether the panel should become focused when it is opened.
     </tr>
+    <tr uxd-api-property name="closeOnEscape" type="boolean" defaultValue="true">
```

Comment:
You could move this up so that it's next to `closeOnExternalClick` since they are conceptually related.

Reply:
No worries